lib/bindings/python/rust/llm/block_manager/vllm/connector/trtllm_leader.rs

```diff
@@ -451,8 +458,17 @@ impl PyTrtllmKvConnectorLeader {
         page_size: usize,
         leader: PyKvbmLeader,
     ) -> Self {
-        let connector_leader: Box<dyn Leader> =
-            Box::new(KvConnectorLeader::new(worker_id, drt, page_size, leader));
+        let metrics_standalone = std::env::var("DYN_KVBM_METRICS_STANDALONE")
+            .map(|v| v == "1" || v.eq_ignore_ascii_case("true"))
```

Comment:
Moving this closure into a function and then calling that function instead of this compound if-statement might be better maintenance-wise. You have this same closure defined ~3 times in this PR and while Rust will do a good job of de-duplicating the closures at compile-time, as a developer I would like to only have to change the code in one spot.